war/src/main/less/abstracts/theme.less

```diff
@@ -169,6 +169,14 @@
   --configure-job-help-area-bg-color: var(--medium-translucent);
 
   // Table
+  --table-background: var(--panel-header-bg-color);
```

Comment:
have you got some working vars for dark theme you could submit a PR for?

Reply:
I have! Might change a little but I've opened a PR - https://github.com/jenkinsci/dark-theme-plugin/pull/165 :)